NEWS.md

```diff
@@ -1,10 +1,20 @@
+# scoringutils 1.2.0
+
+## Package updates
+- The documentation was updated to reflect the recent changes since scoringutils 1.1.0. In particular, usage of the functions `set_forecast_unit()`, `check_forecasts()` and `transform_forecasts()` are now documented in the Vignettes. The introduction of these functions enhances the overall workflow and help to make the code more readable. All functions are designed to be used together with the pipe operator. For example, one can now use something like `data |> set_forceast_unit("location", "model") |> check_forecasts() |> score()`. The function `transform_forecasts()` allows the user to easily add transformations of forecasts, as suggested in the paper ["Scoring epidemiological forecasts on transformed scales"](https://www.medrxiv.org/content/10.1101/2023.01.23.23284722v1). In an epidemiological context, for example, it may make sense to apply the natural logarithm first before scoring forecasts, in order to obtain scores that reflect how well models are able to predict exponential growth rates, rather than absolute values. One can now use something like `data |> transform_forcasts(fun = log) |> score() |> summarise_scores(by = c("model", "scale")`. 
```

Comment:
why not make the code example a working example vs a pseudo example

Reply:
also the code inside `set_forecast_unit` is missing a `c()`.

Reply:
why not use log_shift in the log example?

Reply:
Mostly wasn't sure whether people use NEWS files for that, made it a working example now.

Reply:
Mostly wasn't sure whether people use NEWS files for that, made it a working example now.